### ISSUE TITLE
Commands don't work at all on any server, but they will if I set commands: requireAuth: to false.

### ISSUE THREAD

**Issue by robbiemcleod56** [AuthMe/AuthMeBungee#47]
I have no idea what the problem is, I've done a lot of searching and even went through old issue reports here to find some lead. I have bungee hooks enabled in AuthMeReloaded. 

I am running AuthmeReloaded 5.6.0-SNAPSHOT on my 1.14 server, and 5.4.0 release on all the other servers. 

On my BungeeCord (v Waterfall-Bootstrap:1.14-SNAPSHOT:1f16030:286) instance, I'm running AuthMeBungee 2.2.0-beta1, BungeeChat 2.1.6.855, BungeePing 2.0, BungeeTabListPlus 2.8.10, HubMagic 0.2.6, NuVotifier 2.5.3, and SkinsRestorer 13.6.1. I can give you any logs you need, but where do I start? I've looked through everything I could think was related, and ALL of the 9 servers are having this problem. I have to remove plugin this until the issue is resolved. I only need this to add the FastLogin plugin. 

Another note: In Authme 5.4.0, it is no longer hooks.bungeecord in authme's configuration. It is now this and your instructions on your Spigot page still say the old one. Perhaps this is my problem and I'm overlooking something, but the text seems clear. 

Hooks:
    # Do we need to hook with multiverse for spawn checking?
    multiverse: true
    # Do we need to hook with BungeeCord?
    bungeecord: true
    # Send player to this BungeeCord server after register/login
    sendPlayerTo: ''
    # Do we need to disable Essentials SocialSpy on join?
    disableSocialSpy: false
    # Do we need to force /motd Essentials command on join?
    useEssentialsMotd: false

**Comment by robbiemcleod56** [comment]
Update: It ONLY seems to stop commands on the hub server, and ONLY when joining, whether you're logged in or not. As soon as you switch servers, it allows commands again, and even switching back to hub allows you to enter them again. But leave, join, then try commands, and it won't work. Here is some more of my authme config to help aid you. I think this may have to do with the timeout of a few hours before asking them to log in again. I think there's something wrong when someone is _already_ logged in when joining. 

    sessions:
        # Do you want to enable the session feature?
        # If enabled, when a player authenticates successfully,
        # his IP and his nickname is saved.
        # The next time the player joins the server, if his IP
        # is the same as last time and the timeout hasn't
        # expired, he will not need to authenticate.
        enabled: true
        # After how many minutes should a session expire?
        # A player's session ends after the timeout or if his IP has changed
        timeout: 240

**Comment by robbiemcleod56** [comment]
I feel very embarrassed by this and to have spent so much time trying to figure it out, but I realized as players joined, certain people would complain about not being able to leave the server they entered (no /hub) but everyone in survival was fine and that's where I connected and went back to hub to test if commands would work. Sure enough, only after anyone went on survival specifically, commands could be used across all servers. Seems like a really weird bug, but you MUST get the newest version (at least up to 5.6.0 in my case) for this plugin to function properly at all. I'll close this now, hopefully this thread can help someone else.
